src/mixins/filing-template-mixin.ts

```diff
@@ -55,20 +55,12 @@ export default class FilingTemplateMixin extends DateMixin {
   @Getter getHasPlanOfArrangement!: boolean
   @Getter haveOfficeAddressesChanged!: boolean
   @Getter getCompletingParty!: CompletingPartyIF
-  @Getter isEntityTypeBEN!: boolean
-  @Getter isEntityTypeBC!: boolean
-  @Getter isEntityTypeCCC!: boolean
-  @Getter isEntityTypeULC!: boolean
-  @Getter isEntityTypeCP!: boolean
-  @Getter isEntityTypeSP!: boolean
-  @Getter isEntityTypeGP!: boolean
-  @Getter isBenCorrectionFiling!: boolean
+  @Getter isBaseCorrectionFiling!: boolean
   @Getter isFirmCorrectionFiling!: boolean
   @Getter isClientErrorCorrection!: boolean
   @Getter getAssociationType!: CoopTypes
   @Getter hasAssociationTypeChanged!: boolean
   @Getter getSpecialResolution!: SpecialResolutionIF
-  @Getter isEntityTypeFirm!: boolean
```

Comment:
A bunch of these getters are not used in this mixin, so I moved them to the components that need them.